cli/cmd/inject.go

```diff
@@ -478,6 +478,10 @@ func getOverrideAnnotations(values *charts.Values, base *charts.Values) map[stri
 		overrideAnnotations[k8s.ProxyWaitBeforeExitSecondsAnnotation] = uintToString(proxy.WaitBeforeExitSeconds)
 	}
 
+	if proxy.Await != baseProxy.Await {
+		overrideAnnotations[k8s.ProxyAwait] = strconv.FormatBool(proxy.Await)
```

Comment:
Should this set the annotation to "enabled" or "disabled" instead of "true" or "false"?  Alternatively, we maybe just want to use "true" and "false" for the annotation values.

Reply:
Good catch! Will continue to use `enabled`/`disabled` as values since the annotation values need to be strings.